prime-router/src/main/kotlin/transport/RESTTransport.kt

```diff
@@ -56,7 +57,7 @@ import kotlinx.serialization.json.Json
 import org.json.JSONObject
 import java.io.InputStream
 import java.security.KeyStore
-import java.util.Base64
+import java.util.*
```

Comment:
Better to use explicit imports

Reply:
Good catch. I don't know how that got in there. I'll check my intelliJ settings.